discovery-provider/src/schemas/track_schema.json

```diff
@@ -81,12 +82,14 @@
                 "field_visibility": {
                     "type": "object",
                     "$ref": "#/definitions/FieldVisibility",
-                    "default": null
+                    "default": null,
+                    "JSONDefault": "definitions.FieldVisibility.properties"
                 },
                 "download": {
                     "type": "object",
                     "$ref": "#/definitions/Download",
-                    "default": null
+                    "default": null,
+                    "JSONDefault": "definitions.Download.properties"
                 },
                 "stem_of": {
```

Comment:
stem_of and remix_of should also be getting smarter defaults? or are these nullable

Reply:
these are nullable now!